fastestimator/backend/_focal_loss.py

```diff
@@ -53,7 +99,8 @@ def focal_loss(y_true: Tensor, y_pred: Tensor, gamma: float = 2.0, alpha: float
                 positive vs negative examples or (-1/None) to ignore. Default = 0.25
         gamma: Exponent of the modulating factor (1 - p_t) to
                balance easy vs hard examples.
-        from_logits: Whether y_pred is logits (without softmax).
+        normalize: Whether to normalize along the batch size.
```

Comment:
"y_true: Ground truth class labels which should take values of 1 or -1."

I believe y_true should be 0 or 1 right? also we need to let user know the shape too